chain/x/zoracle/internal/types/report.go

```diff
@@ -39,3 +39,17 @@ func NewValidatorReport(
 		Validator:  valAddress,
 	}
 }
+
+// ExternalData is a struct that capsulized external data id and raw data
+type ExternalData struct {
```

Comment:
ExternalData encapsulates a raw data report for an external data source from a block validator.